example_new/KevinHall.json

```diff
@@ -24,33 +24,28 @@
     ],
     "Foods": [
         {
-            "Name": "Alcohol",
-            "Nutrients": {},
-            "Price": null
-        },
-        {
-            "Name": "Carbohydrate",
+            "Name": "FoodCarbohydrate",
             "Nutrients": {
                 "Carbohydrate": 1.0
             },
             "Price": null
         },
         {
-            "Name": "Fat",
+            "Name": "FoodFat",
```

Comment:
Why are you prefixing things with food?

Reply:
To avoid confusion with 'actual' nutrients in the risk factor mapping. These atre just dummy/meta foods, where the 'food' protein provides 100% 'real' protein, and nothing else.

Reply:
To avoid confusion with 'actual' nutrients in the risk factor mapping. These are just dummy/meta foods, where i.e. the 'food' protein provides 100% 'real' protein, and nothing else.

Reply:
To avoid confusion with 'actual' nutrients in the risk factor mapping. These are just dummy/meta foods, where i.e. the 'food' protein provides 100% 'real' protein, and nothing else.